# Shradha_Mukherjee_Resume

Shradha Mukherjee 
PhD in Biochemistry                    		            
MS in Chemistry                                                                                
MAS in Health Informatics                                                                                           


Email: smukher2@gmail.com                                                                                                              
Github: https://github.com/smukher2
NCBI Publication: https://www.ncbi.nlm.nih.gov/pubmed/?term=Shradha+Mukherjee                                                
                                                  
 
## Summary                 
                                                                                                                              
 Health informatics and bioinformatics expert, with extensive experience in neuroscience, stem cell biology, neurotoxicity, neurodegeneration, cancer and chemistry.

 Innovative analytical thinker adept in application of interdisciplinary experimental and computational approaches to challenges in healthcare.

 Excellent communication skills as evidenced by oral and poster presentations at international conferences which lead to collaborations, publications, citations and grant funding.


## Previous Employment
	
 Postdoctoral Fellow, University of Texas Southwestern Medical Center, Dallas, TX, U.S. in the laboratory of Dr. Jenny Hsieh                                        2011-2017 

	• Designed and utilized ‘omics’ approaches, which lead to discovery of a novel role of ribosome biogenesis genes in regulation of adult neural stem cells. 

	• Effectively utilized cutting edge molecular biology techniques of gain-of-function and loss-of-function experiments to validate role of ribosome biogenesis genes in  
	  regulation of adult stem cell quiescence. 

 Staff scientist (part-time ≥50%, technician) at University of California Los Angeles, U.S.                                                                      2018-Feb2021


## Education

 PhD (Doctorate) in Biochemistry, University of Houston, Houston, TX, U.S. in the laboratory of Dr. Donald A. Fox                                                   2005-2011 

	• Contributed in analysis of Affymetrix gene expression arrays, which resulted in discovery of genes underlying toxic lead (Pb) induced developmental delay in retina.

	• Deciphered spatio-temporal expression of developmental genes and changes in spatio-temporal pattern of expression of these genes induced by lead (Pb). 

 MAS (Advanced MS) in Health Informatics Arizona State University, Tempe, AZ, U.S.                                                                                  2017-2019 
  
 MS (Masters in Science) in Chemistry, Indian Institute of Technology (IIT), Madras, India.                                                                         2003-2005 
                                                                                                                                   

## Other Experiences: Independent Computational Research Projects 

 Independent Researcher      	                                                                                                                                     2017-2020 
From 2017-2020, I did several other independent computational research projects, using free open access NIH/NSF resources, personal resources and personal funds. These projects are completed and published or accepted for publication and are self-owned (not affiliated with any organization/university). Cancer Biotherapy and Radiopharmaceuticals, and Scientific Reports journals generously supported publication of these projects in their journals. To drive discovery and innovation all protocols/computational pipelines https://github.com/smukher2 from these projects have been made open source and customizable so that it can be used in other studies. 

	• Developed an integrated computational pipeline for identification of common molecular markers of age-related neurodegenerative diseases (Alzheimer’s disease, 	  Parkinson’s disease, Huntington’s disease) and Multiple Sclerosis. 

	• Discovered a novel set of stem cell markers that can distinguish between Glioblastoma and normal tissue using a systems biology computational approach.

 Remotely conducted applied project for Advanced Masters Researcher (MAS), Arizona State University, AZ, U.S.                                                       2019 Fall

	• Developed a computational pipeline to improve interoperability of drug-gene interaction and pathogenic variant databases, for potential use in precision medicine 	      treatment of Non-Small Cell Lung Cancer (NSCLC) patients. 

	• Expanded the above computational pipeline to create computable drug-gene interaction BEL (Biological Expression Language, https://language.bel.bio/) statements, for 	         visual display of these interactions on BioDati Studio.Inc https://biodati.com/.


## Other Experiences: Editorial/Reviewer Roles and Professional Memberships 

 Served as reviewer on several scientific publications, such as Cancer Biotherapy and Radiopharmaceuticals https://www.liebertpub.com/loi/cbr , Scientific Reports             https://www.nature.com/srep/ and others.                                                                                                                        2017-present

 Served as invited Editor of new journal from Imedpub, Journal of Cell and Development Biology http://www.imedpub.com/journal-cell-developmental-biology/editors.php to         promote research that deciphers the link between basic biology and disease.                                                                                       2017-2018 
  
 Served as vice-president/board member of University of Texas Southwestern Medical Center, TX, U.S. postdoctoral association (over 550 members). During this time also served   as editor and contributing author for several issues the association’s newsletter.                                                                                 2011-2015 

 Current member of American Association for the Advancement of Science (AAAS) https://www.aaas.org/

 Current member of Society for Neuroscience (SFN) https://www.sfn.org/

 Current community member of American Chemical Society (ACS) https://www.acs.org

 Current member of American Society for Cell Biology (ASCB) https://www.ascb.org/


## Certifications

 Certification on ‘Becoming a more equitable educator: Mindsets and Practices’, MITx MicroMasters, MIT open learning, U.S.                                               2020

 ‘Managing the company of the future’: University of London, London, UK                                                                                                  2019 

 ‘Responsible Conduct of Research Involving Humans’: University of California Los Angeles, CA, U.S.                                                                      2018 

 ‘Genomic Data Science’: Johns Hopkin’s University, Baltimore, MD, U.S.                                                                                                  2017

 ‘Data or specimen only research’, human data: Collaborative Institutional Training Initiative (CITI), U.S.                                                              2017 


## Teaching and Mentoring

 Mentored high school, undergraduate, graduate, postdoctoral researchers and medical students in molecular biology and bioinformatics skills at University of Texas           	 Southwestern Medical Center, Dallas, TX, U.S.                                                                                                                      2011-2017 

 Attended “Entering Mentoring” workshop series by UT Southwestern Career Development and Postdoctoral Office, based on workshops by National Research Mentoring Network.  	                                                                                                                                                                2016-2017 

 Helped to develop and lecture “Introduction to Biology” lab course to enhance learning experience for undergraduate students (30 students per class) at University of      	Houston, Houston, TX, U.S.                                                                                                                                        2005-2006                


## Computational Skills

 Artificial intelligence and Machine Learning

 Next generation sequencing (RNA-seq, ChIP-seq) analysis and large dataset management

 Cluster computing and cloud computing (Cyverse, Galaxy) 

 Statistical, bioinformatics and informatics programming in scripting languages (R, UNIX, Python and SQL)

 Variant analysis (FreeBayes, GATK)

 Public databases in genomics and health informatics (IVY GAP, GTEx, TCGA, NCBI-GEO, EMBL-EBI, PhysioNET, MIMIC etc.) 

 Bioinformatics tools and analysis pipelines (WGCNA, SVA, Homer, DESeq, edgeR, Samtools, Bioconductor, Cuffdiff, GSEA, NIH-DAVID, PANTHER etc.)


## Experimental Skills

 Molecular biology techniques (cDNA cloning, shRNA cloning, electroporation, transduction)

 Cell biology and biochemistry techniques (Western blot, qPCR, co-immunoprecipitation)

 Confocal, fluorescent microscopy and flow cytometry

 Animal surgery (brain, retina)


## Awards

 Awarded scholarship for data science at Data Incubator https://www.thedataincubator.com/ and 12 day data science online primer course (top 2to3% applicants are selected)                                                                                                                                                                             2020 
                                       
 Awarded scholarship for Genomic Data Science certificate bioinformatics course and capstone project by Johns Hopkins University, Baltimore, MD, U.S. certificate course.                                                                                                                                                                               2017 
                                                       
 Awarded scholarship for bioinformatics project and invited to summer school in systems genetics of neurodegeneration, Frauenchiemsee, Germany.                          2017                                          
                                                                             		                       
 Awarded travel award for presentation of discoveries on molecular regulation of stem cells using cell biology and bioinformatics techniques by Keystone Symposia Orange County, CA, U.S.                                                                                                                                                    2014, 2017 

 Cutaneous and Ocular Toxicology Travel Grant Award Association for Research in Vision and Ophthalmology, for presentation on lead (Pb) toxicity in retina Fort Lauderdale, FL, U.S.                                                                                                                                                                  2009 

 
## Select Publications †first author *corresponding author
Articles as listed on PubMed https://www.ncbi.nlm.nih.gov/pubmed/?term=Shradha+Mukherjee
and open access link to full text of articles are provided below.

 Mukherjee S*† (under review, work started in 2017). Immune signature genes are enriched in shared gene networks associated with neurological disorders: Multiple Sclerosis (MS), Alzheimer’s disease (AD), Parkinson’s disease (PD) and Huntington’s disease (HD). (Independent research).

 Mukherjee S*† (2020). Quiescent stem cell marker genes in glioma gene networks are sufficient to distinguish between normal and glioblastoma (GBM) samples. Scientific Reports 10, 10937. https://doi.org/10.1038/s41598-020-67753-5 (Independent research).

 Mukherjee S*†, Klaus C†, Pricop-Jeckstadt M, Miller JA and Struebing FL*† (2019). A Microglial Signature Directing Human Aging and Neurodegeneration-Related Gene Networks. Frontiers in Neuroscience 13:2. https://doi.org/10.3389/fnins.2019.00002 (Independent research).

 Mukherjee S*† (2019). Genomics-Guided Immunotherapy for Precision Medicine in Cancer. Cancer Biotherapy and Radiopharmaceuticals. 10.1089/cbr.2018.2758. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6802731/  (Independent research).

 Mukherjee S†, Hsieh J* (2018). Genome-Wide Identification of Transcription Factor-Binding Sites in Quiescent Adult Neural Stem Cells. In: Lacorazza H. (eds) Cellular Quiescence. Methods in Molecular Biology, vol 1686. Humana Press, New York, NY (Springer Nature). https://pubmed.ncbi.nlm.nih.gov/29030827/

 Mukherjee S†, Brulet R, Zhang L and Hsieh J* (2016). REST regulation of gene networks in adult neural stem cells. Nature Communications 7:13360. https://doi.org/10.1038/ncomms13360

 Chaney S, Mukherjee S, Giddabasappa A, Rueda R, Hamilton R, Johnson JE and Fox DA* (2016). Increased proliferation of late-born retinal progenitor cells by gestational lead exposure delays rod and bipolar cell differentiation. Molecular Vision 22:1468. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5204453/

 Mukherjee S† and Hsieh J* (2013). On your (methyl) mark, get TET1, go! Cell Stem Cell Preview 13(2):133. https://doi.org/10.1016/j.stem.2013.07.011

 Giddabasappa A, Hamilton R, Chaney S, Xiao W, Johnson JE, Mukherjee S, Fox DA (2011). Low-level gestational lead exposure increases retinal progenitor cell proliferation and rod photoreceptor and bipolar cell neurogenesis in mice. Environmental Health Perspectives 119: 71-77. https://doi.org/10.1289/ehp.1002524

 Fox DA, Hamilton R, Johnson JE, Xiao W, Chaney S, Mukherjee S, Miller D, O’Callaghan JP (2011) Gestational lead exposure selectively decreases retinal dopamine amacrine cells and dopamine content in Adult Mice. Toxicology and Applied Pharmacology 256(3):258-257. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205288/
